---
string File
string Action
bool ActionDefined
